sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -0,0 +1,741 @@
+    createdDateTime: Date;
+    lastUpdatedDateTime: Date;
+    status: OperationStatus;
+}
+
+// @public (undocumented)
+export type AnalyzeOptions = ExtractReceiptOptions | ExtractLayoutOptions | ExtractCustomFormOptions;
+
+// Warning: (ae-forgotten-export) The symbol "ICanHazStatus" needs to be exported by the entry point index.d.ts
+//
+// @public
+export type AnalyzePollerClient<T extends ICanHazStatus> = {
+    startAnalyze: (body: HttpRequestBody, contentType: SupportedContentType, analyzeOptions: AnalyzeOptions, modelId?: string) => Promise<{
+        operationLocation: string;
+    }>;
+    getAnalyzeResult: (resultId: string, options: {
+        abortSignal?: AbortSignalLike;
+    }) => Promise<T>;
+};
+
+// @public
+export type AnalyzeReceiptOperationResult = Omit<AnalyzeOperationResultModel, 'analyzeResult'> & {
+    analyzeResult?: AnalyzeReceiptResult;
+};
+
+// @public
+export type AnalyzeReceiptResult = Omit<AnalyzeResult, 'documentResults'> & {
+    receiptResults?: ReceiptResult[];
+};
+
+// @public
+export type AnalyzeReceiptResultResponse = AnalyzeReceiptOperationResult & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: AnalyzeOperationResultModel;
+    };
+};
+
+// @public (undocumented)
+export type AnalyzeResult = Omit<AnalyzeResultModel, "pageResults"> & {
+    pageResults?: PageResult[];
+};
+
+// @public
+export interface AnalyzeResultModel {
+    documentResults?: DocumentResult[];
+    errors?: ErrorInformation[];
+    pageResults?: PageResultModel[];
+    readResults: ReadResult[];
+    version: string;
+}
+
+// @public (undocumented)
+export type ArrayFieldValue = {
+    type: "array";
+    valueArray: FieldValue[];
+} & CommonFieldValue;
+
+// @public
+export class CognitiveKeyCredential implements ServiceClientCredentials {
+    constructor(apiKey: string);
+    signRequest(webResource: WebResource): Promise<WebResource>;
+    updateKey(apiKey: string): void;
+}
+
+// @public (undocumented)
+export interface CommonFieldValue {
+    boundingBox?: number[];
+    confidence?: number;
+    elements?: TextElement[];
+    page?: number;
+    text?: string;
+}
+
+// @public (undocumented)
+export type CustomFormModel = Omit<Model, "trainResult"> & {
+    kind: "unlabeled";
+    trainResult?: CustomFormModelTrainResult;
+};
+
+// @public (undocumented)
+export type CustomFormModelResponse = CustomFormModel & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: Model;
+    };
+};
+
+// @public (undocumented)
+export type CustomFormModelTrainResult = Omit<TrainResult, "averageModelAccuracy" | "fields">;
+
+// @public
+export class CustomFormRecognizerClient {
+    constructor(endpointUrl: string, credential: TokenCredential | CognitiveKeyCredential, options?: FormRecognizerClientOptions);
+    // (undocumented)
+    deleteModel(modelId: string, options?: DeleteModelOptions): Promise<import("@azure/core-http").RestResponse>;
```

Comment:
strange to see an inline import here..

Reply:
also not sure why it's a raw RestResponse

Reply:
I missed specifying return type for this.